src/js/json_include/index.js

```diff
@@ -42,7 +42,7 @@ export class JSONInclude {
                     });
                 }
             }
-            if (isIncludeExp) return;
+            // if (isIncludeExp) return;
```

Comment:
We should either remove this (and explain why we don't need it) or put a "TODO" item explaining what's needed to be done in the future to uncomment this.

Reply:
I removed it because this logic prevented us from parsing this example:
```
{
    "...": "include(lattice/lattice_bravais.json)",
    "name": "lattice",
    "vectors": {
        "...": "include(lattice/lattice_vectors.json)"
    }
}
```
Before my changes, it could resolve only the top line `"...": "include(lattice/lattice_bravais.json)"` without `"...": "include(lattice/lattice_vectors.json)"` 
This is actually the first case with multiple `include` in examples folder, that's why previously everything worked fine